namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-template-typescript/resources/elasticache.tf

```diff
@@ -3,18 +3,20 @@
 ################################################################################
 
 module "elasticache_redis" {
-  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=5.5"
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=5.6"
   vpc_name               = var.vpc_name
+  team_name              = var.team_name
+  business-unit          = var.business_unit
   application            = var.application
-  environment-name       = var.environment
   is-production          = var.is_production
-  infrastructure-support = var.infrastructure_support
-  team_name              = var.team_name
-  number_cache_clusters  = var.number_cache_clusters
-  node_type              = "cache.t2.small"
-  engine_version         = "4.0.10"
-  parameter_group_name   = "default.redis4.0"
   namespace              = var.namespace
+  environment-name       = var.environment
+  infrastructure-support = var.infrastructure_support
+
+  number_cache_clusters = var.number_cache_clusters
+  node_type             = "cache.t4g.small"
+  engine_version        = "6.x"
+  parameter_group_name  = "default.redis6.x"
```

Comment:
instance class `t2` → `t4g` because it's cheaper and offers better performance
redis version `4` → `6` because 4 has probably reached [end of life](https://redis.io/docs/about/releases/)

NB: t4g does not support redis 4 so this upgrade might not be possible in 1 step…? this namespace is just a demo so the data can be destroyed and downtime is not a problem